bevy_replicon_example_backend/examples/tic_tac_toe.rs

```diff
@@ -595,12 +613,27 @@ struct Cell {
     index: usize,
 }
 
-/// Contains player ID.
-///
-/// We want to replicate all players, so we just set [`Replicated`] as a required component.
-#[derive(Clone, Copy, Component, Deref, Serialize, Deserialize)]
+/// Marker for a player entity.
+#[derive(Component, Default)]
 #[require(Replicated)]
-struct Player(ClientId);
+struct Player;
+
+/// Marks [`Player`] as locally controlled.
+///
+/// Used to determine if player can place a symbol.
+///
+/// See also [`local_player_turn`].
+#[derive(Component)]
+#[require(Player)]
+struct LocalPlayer;
+
+/// Identifies which player controlled by which client.
+///
+/// Points to client entity. Used to verify client's turns.
+///
+/// It's not replicated and present only on server or singleplayer.
+#[derive(Clone, Copy, Component, Deref)]
+struct PlayerClient(Entity);
```

Comment:
I reworked examples to avoid replicating components identifiers.
Client don't need know them and it's extra traffic. So we use a separate component that associates spawned entities with connected clients.